plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -489,8 +480,7 @@ def __init__(self, ansible_module):
 
         def __getitem__(self, key):
             param = self.mapping[key]
-            if param is not None:
-                return _afm_convert(param)
+            return _afm_convert(param) if param is not None else None
```

Comment:
Please change it, it is hard to read. What about `return None if param is None else _afm_convert(param)`